backend/app.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser")
+const time = require("express-timestamp")
+
+const School = require("./School")
+
+port = 5000;
+const app = express();
+app.use(bodyParser.urlencoded({extended: false}))
+app.use(express.json());
+app.use(cors())
+app.use(time.init)
+
+let mySchool = new School();
+console.log(mySchool)
+console.log(mySchool)
+mySchool.enrollStudent("algebra")
+console.log(mySchool)
+mySchool.getStudentsByClass("algebra")
+
+app.get('/', (req, res) => {
+    console.log('youve hit GET/ endpoint', mySchool)
+    res.json({
+        message: 'youve hit GET/ endpoint',
+        school: mySchool,
+    })
+})
+
+app.post("/class", (req,res) => {
+    mySchool.addClass(req.body.classname, req.body.teacher)
```

Comment:
Assign this to a variable, let's say `let newClass = mySchool.addClass(req.body.classname, req.body.teacher)` so you can include this in your response